### PR TITLE
PR updates

### PR DESCRIPTION
**Pull request by rkeilty**
Looks good overall, no major changes!

### CODE REVIEW

File: docker/compose/nginx-flask-mysql.yml [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
+      MYSQL_DATABASE: todo_app
+      MYSQL_USER: web_user
+      MYSQL_PASSWORD: password
+
+  nginx:
+    image: brianmorgan/nginx:latest
+    depends_on:
+      - web
     environment:
-      - UWSGI_HTTP_PORT=5000
```
Comment (rkeilty) [inline docker/compose/nginx-flask-mysql.yml:15]: Prefer block style over flow style (seems more standard across other `compose` files.)

File: docker/flask-uwsgi/Dockerfile [hunk 2]
```diff
@@ -1,13 +1,16 @@
 FROM ubuntu:xenial
 
-ENV PYTHONUNBUFFERED 1
```
Comment (rkeilty) [inline docker/flask-uwsgi/Dockerfile:3]: To save some weird output, ensure we run in non-interactive mode.

File: docker/compose/nginx-flask-mysql.yml (hunk 1 above)
Comment (bmorgan21) [reply docker/compose/nginx-flask-mysql.yml:15]: what does that mean?

Comment (rkeilty) [reply docker/compose/nginx-flask-mysql.yml:15]: It's just in reference to the method by which you can structure the `yaml`.  You can do `- UWSGI_HTTP_PORT=5000` or `UWSGI_HTTP_PORT: 5000` which is better picked up by syntax checkers.

**Pull request merged**